plugins/doc_fragments/gitlab.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+class ModuleDocFragment(object):
+
+    # Standard files documentation fragment
+    DOCUMENTATION = r'''
+requirements:
+  - requests (python library https://github.com/psf/requests)
+
+options:
+  api_token:
+    description:
+      - GitLab token for logging in.
```

Comment:
I personally like this description better (it is used in a similar amount of modules):
```suggestion
      - GitLab access token with API permissions.
```
What do you think?

Reply:
Will change